affectations.py

```diff
@@ -0,0 +1,140 @@
+#! /usr/bin/env python3
+
+import math
+import random
+
+
+def main():
+
+    apprentis = []
+    mentors = []
+
+    # Creating two lists, of mentors and apprentis respectively
+    with open('members.txt', 'r') as f:
```

Comment:
Le fichier itère par défaut sur les lignes.  Tu peux donc faire `for line in f:`.  Tu peux aussi simplifier en commencent chaque itération par `line = line.strip()` pour enlever les espaces et les changements de lignes.

Reply:
adopté!